dashboard/config/locales/data.en.yml

```diff
@@ -178,7 +178,7 @@ en:
         'bounce_12_basketball': "Create your own Basketball game. You can change all the rules. When you're done, click Finish to let friends try your game on their phones."
         'bounce_1_sports': "Get to the flag! Attach a block to the \"when left arrow\" block and press \"Run\" to move the player to the flag."
         'bounce_2_sports': "There are two flags now! Attach a block to the \"when left arrow\" and \"when right arrow\" blocks to make your player move in two directions."
-        'bounce_5_sports': "The hockey puck is going to fly down toward the player! Attach a block to the \"when ball hits hand\" block to ensure the ball reaches the flag."
+        'bounce_5_sports': "The hockey puck is going to fly down toward the player! Attach a block to the \"when ball hits player\" block to hit the flag."
```

Comment:
Yep, this was really the "respond to all of Ryan's comments" PR. Block titles were just the biggest change.